### PR TITLE
Added meta-qt5 support

### PR DESCRIPTION
**Pull request by atomass**
As previously discussed with @jan-kiszka ( #85 ), I've added support to meta-qt5 layer.
I hope I have chosen the right way of doing this. Otherwise, don't hesitate to point me in the right direction.

Also, fixed typos in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ $ kas build meta-iot2000-bsp/kas.yml
 You can also reproduce the Windows or Linux SDK this way:
 
 ```shell
-$ kas build meta-iot2000-example/sdk-kas-windows-i586.yml
-$ kas build meta-iot2000-example/sdk-kas-linux-x86.yml
+$ kas build meta-iot2000-example/kas-sdk-windows-i586.yml
+$ kas build meta-iot2000-example/kas-sdk-linux-x64.yml
```
Comment (jan-kiszka) [inline README.md:74]: Good catch, but we should split this up into a separate commit.

File: meta-iot2000-bsp/recipes-devtools/qt5/iot2000-example-image.bbappend [hunk 2]
```diff
@@ -0,0 +1 @@
+inherit populate_sdk_qt5
```
Comment (jan-kiszka) [inline meta-iot2000-bsp/recipes-devtools/qt5/iot2000-example-image.bbappend:1]: Style: please add a new line at the end here.

Does this populate the SDK with QT bits even if the user does not include QT? My first guess would be "yes", and that would not be desirable.

**Review by jan-kiszka — CHANGES_REQUESTED** [review]
First of all, sorry for the late reply.

Besides the inline comments, please also sign off your commit according to https://developercertificate.org (see "signed-off-by" tags in other commits).

File: README.md (hunk 1 above)
Comment (atomass) [reply README.md:74]: Ok, I created a new pull request for this file only

File: meta-iot2000-bsp/recipes-devtools/qt5/iot2000-example-image.bbappend (hunk 2 above)
Comment (atomass) [reply meta-iot2000-bsp/recipes-devtools/qt5/iot2000-example-image.bbappend:1]: Sorry, I didn't realize could lead to that. Maybe we can put `INHERIT += " populate_sdk_qt5"`
somewhere, in `qt5-iot2000.inc` perhaps?! But I think it only works inside configuration files.
This is only needed in order to export qt5 libraries during sdk creation.

**Comment by jan-kiszka** [comment]
As we have no solution yet how to make this dependency truly optional, I'm closing the PR. Can be reopened with a new version at any time.
